README.md

```diff
@@ -4,7 +4,7 @@
 
 Are you prepared to embark on your quest to receive mentoring and land a job? You've arrived exactly where you need to be. Follow the roadmaps, educate yourself, and then apply for positions.
 
-This repository contains the list of communities and job portals you can join and apply to. You can also add your know job and other portals and help 
+This repository contains the list of communities and job portals you can join and apply to. You can also add your current job and other portals and help 
```

Comment:
I think `You can also add your known job ...` would sound much better than `current`. Also 
- please create an issue and get it assigned to you
- Link this pr to that issue

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix